src/reducers/index.spec.js

```diff
@@ -39,304 +35,390 @@ describe('reducers', function() {
         time: {
           hh: 1,
           mm: 10,
           zone: 'UTC'
         },
-        halted: false
-      }
-    };
-    const newState = lazyJar(state, action);
-    assert.deepEqual(expected, newState);
-  });
+        halted: true
+  }
+  const newState = lazyJar(state, action);
+  assert.deepEqual(expected, newState);
+});
 
-  it('should correctly remove project from state with TERMINATE action', function() {
-    const state = {
-      'lazy-jar': {
-        userIds: [0, 2],
-        frequency: 'WEEKDAYS',
-        time: {
-          hh: 1,
-          mm: 10,
-          zone: 'UTC'
+it('should correctly update the state given START action', function () {
+  const state = {
+      team_id: 'CBV',
+      event_id: 'lazy-jar',
+      time_to_respond: 900,
+      members: [{
+          user_id: 2,
+          ignore: false,
+          skip_until: '16-10-2018'
         },
-        halted: false
+        {
+          user_id: 3,
+          ignore: false
+        }
+      ],
+      frequency: 'WEEKDAYS',
+      time: {
+        hh: 1,
+        mm: 10,
+        zone: 'UTC'
       },
-      artris: {
-        userIds: [0, 2],
-        frequency: 'WEEKDAYS',
-        time: {
-          hh: 1,
-          mm: 10,
-          zone: 'UTC'
-        },
-        halted: false
-      }
-    };
-    const action = {
-      type: TERMINATE,
-      event: 'artris'
-    };
-    const expected = {
-      'lazy-jar': {
-        userIds: [0, 2],
-        frequency: 'WEEKDAYS',
-        time: {
-          hh: 1,
-          mm: 10,
-          zone: 'UTC'
-        },
-        halted: false
+      halted: true
+  }
+  const action = {
+    type: START,
+    event: 'lazy-jar',
+    userId: 2,
+  };
+  const expected = {
+      team_id: 'CBV',
+      event_id: 'lazy-jar',
+      time_to_respond: 900,
+      members: [{
+        user_id: 2,
+        ignore: false,
+      }, {
+        user_id: 3,
+        ignore: false
+      }],
+      frequency: 'WEEKDAYS',
+      time: {
+        hh: 1,
+        mm: 10,
+        zone: 'UTC'
+      },
+      halted: true
+  }
+  const newState = lazyJar(state, action);
+  assert.deepEqual(expected, newState);
+});
+
+it('should correctly update the state given STOP action', function () {
+const state = {
+  team_id: 'CBV',
+    event_id: 'lazy-jar',
+    time_to_respond: 900,
+    members: [{
+        user_id: 2,
+        ignore: false,
+        skip_until: '16-10-2018'
+      },
+      {
+        user_id: 3,
+        ignore: false
       }
-    };
-    const newState = lazyJar(state, action);
-    assert.deepEqual(expected, newState);
-  });
+    ],
+    frequency: 'WEEKDAYS',
+    time: {
+      hh: 1,
+      mm: 10,
+      zone: 'UTC'
+    },
+    halted: true
+}
+const action = {
+  type: STOP,
+  event: 'lazy-jar',
+  userId: 2,
+};
+const expected = {
+    team_id: 'CBV',
+    event_id: 'lazy-jar',
+    time_to_respond: 900,
+    members: [{
+      user_id: 2,
+      ignore: true,
+      skip_until: '16-10-2018'
```

Comment:
not an issue but a preference, if the `skip_until` is meaningless after the STOP command, I think we should delete it from the state